### PR TITLE
Make Citations, titles and, descriptions appear in markdown in backmatter

### PR DESCRIPTION
**Pull request by schen2166**
The Backmatter has markup language shown in prose in the screenshots below. Wrap them in a `OSCALMarkupProse` component so that it is easier for users to read.
<br/>

### CODE REVIEW

**Comment by schen2166** [comment]
This screenshot shows that the Title, and citation descriptions have been updated to be shown in markdown.

File: src/components/OSCALBackMatter.js [hunk 1]
```diff
@@ -47,7 +48,9 @@ function DescriptionDisplay(props) {
     );
   }
   return (
-    <StyledTooltip title={props.resource.description}>
+    <StyledTooltip
+      title={<OSCALMarkupLine text={props.resource.description} />}
```
Comment (kylelaker) [inline src/components/OSCALBackMatter.js:52]: `description` is `markup-multiline`: https://pages.nist.gov/OSCAL/reference/latest/catalog/json-reference/#/catalog/back-matter/resources/description

**Review by kylelaker — CHANGES_REQUESTED** [review]

File: src/components/OSCALBackMatter.js [hunk 2]
```diff
@@ -31,7 +32,7 @@ function TitleDisplay(props) {
   const color = props.resource.title ? "initial" : "error";
   return (
     <Typography color={color} variant="subtitle1">
-      {title}
+      <OSCALMarkupLine text={title} />
```
Comment (tuckerzp) [inline src/components/OSCALBackMatter.js:36]:
```suggestion
      <OSCALMarkupLine>{title}</OSCALMarkupLine>
```

Comment (kperk103) [reply src/components/OSCALBackMatter.js]: The most recent commit addresses this

**Review by kylelaker — APPROVED** [review]

**Review by Bronstrom — APPROVED** [review]

**Pull request merged**